background.js

```diff
@@ -152,11 +153,16 @@ ipcMain.on('k8s-state', (event) => {
 ipcMain.on('k8s-reset', async(event, arg) => {
   try {
     // If not in a place to restart than skip it
-    if (![K8s.State.STARTED, K8s.State.READY, K8s.State.STOPPED].includes(k8smanager.state)) {
+    if (![K8s.State.STARTED, K8s.State.READY, K8s.State.STOPPED, K8s.State.ERROR].includes(k8smanager.state)) {
       console.log(`Skipping reset, invalid state ${ k8smanager.state }`);
 
       return;
     }
+    if (k8smanager.version !== cfg.kubernetes.version) {
+      // When changing versions, we always need to do a slow reset to recreate
+      // the cluster.
+      arg = 'slow';
+    }
```

Comment:
Is this required? Once we have kim available for image management the images will be stored in the VM. Blowing it away will be problematic.

Reply:
It is required for minikube use, because that lays down files (the kubelet systemd unit files) that are tied to versions; once we switch to directly using hyperkit we could look again.

Reply:
I just tested and realized that.

I also found I'm seeing brief errors because of unhandled homestead promise rejectsions.